content/blog/super-charging-our-github-action-with-the-pulumi-automation-api/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "GitHub Action now with added Pulumi Automation API"
+h1: "Super Charging our GitHub Action with the Pulumi Automation API"
+authors: 
+  - paul-stack
+#  - simen-olsen
+tags: 
+  - GitHub Actions
+  - automation api
+date: "2021-02-24"
+#meta_image: ""
+meta_desc: "Simen Olsen teams up with Paul Stack to introduce the new GitHub Action that has been super charged with the Pulumi Automation API"
+---
+
+**Guest Article:** <About Simen here>
+
+In this article, we'll take a look at how our [GitHub Action](https://github.com/pulumi/actions) has been re-architected to be built on top of our [Automation API]({{< relref "/blog/automation-api" >}}).
+
+<!--more-->
+
+The Pulumi GitHub Action was created back in 2018. This action was a [docker based GitHub Action](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action) which
+allowed us to be able to package up all the necessary pre-requisities and the Pulumi CLI to allow users to be able to
+run Pulumi commands. As the Pulumi ecosystem has grown, so too did the container that GitHub Actions run.
+
+The alternative way to create a GitHub Action is to create a [JavaScript based action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action).
```

Comment:
```suggestion
Since then, an alternative way to create a GitHub Action has become available, [by simply writing JavaScript](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action). At Pulumi, and in our community, we love expressing everything we possibly can in code, so this new support got us quite excited about the "better together" possibilities!
```